src/component/framework/form-group.js

```diff
@@ -31,7 +37,7 @@ export class FormGroup {
    */
   @computedFrom('element')
```

Comment:
it does. and since the element doesn't change (i assume since its a reference), but only eg element.key , i wasn't sure if that change would be detected on element. but if you say it does, then it does